### PR TITLE
Checking attestors for duplicates

### PR DESCRIPTION
**Pull request by ChaosInTheCRD**
A quick recreation of https://github.com/in-toto/witness/pull/332 as some DCO signoff was missing.

### CODE REVIEW

**Review by jkjell — COMMENTED** [review]
Looks good so far. Could we add a test in https://github.com/in-toto/witness/blob/main/cmd/run_test.go? I think we're good then.

**Comment by ChaosInTheCRD** [comment]
> Looks good so far. Could we add a test in https://github.com/in-toto/witness/blob/main/cmd/run_test.go? I think we're good then.

Done!

File: docs/witness_run.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ witness run [cmd] [flags]
 
 ```
       --archivista-server string                      URL of the Archivista server to store or retrieve attestations (default "https://archivista.testifysec.io")
-  -a, --attestations strings                          Attestations to record (default [environment,git])
+  -a, --attestations strings                          Attestations to record (default [product,material])
```
Comment (jkjell) [inline docs/witness_run.md:13]: You'll have to change https://github.com/in-toto/witness/blob/main/options/run.go#L40 and run `make docgen`

File: docs/witness_sign.md [hunk 2]
```diff
@@ -22,6 +22,7 @@ witness sign [file] [flags]
   -k, --signer-file-key-path string                   Path to the file containing the private key
       --signer-fulcio-oidc-client-id string           OIDC client ID to use for authentication
       --signer-fulcio-oidc-issuer string              OIDC issuer to use for authentication
+      --signer-fulcio-oidc-redirect-url string        OIDC redirect URL (Optional). The default oidc-redirect-url is 'http://localhost:0/auth/callback'.
```
Comment (jkjell) [inline docs/witness_sign.md:25]: Same comment as above.

Comment (ChaosInTheCRD) [reply docs/witness_sign.md:25]: This is fine right? I think i'm correct in saying that this change to the docs is a consequence of `go-witness` getting bumped.

Comment (jkjell) [reply docs/witness_sign.md:25]: Looks that way since the make docsgen check passed. 😅

**Review by jkjell — APPROVED** [review]

**Pull request merged**